admin/src/components/CMEditViewTranslateLocale/index.js

```diff
@@ -131,41 +132,48 @@ const Content = ({
       return
     }
 
-    const requestDataURL = `/content-manager/collection-types/${slug}/${value}`
     const translateURL = `/deepl/translate`
 
     setIsLoading(true)
     try {
-      const { data: response } = await axiosInstance.get(requestDataURL)
-
-      const cleanedData = cleanData(response, allLayoutData, localizations)
-
       const { locale: sourceLocale } = localizations.find(
         ({ id }) => id == value
       )
       const { data: translatedData } = await axiosInstance.post(translateURL, {
-        data: cleanedData,
+        id: value,
         sourceLocale,
         targetLocale: currentLocale,
         contentTypeUid: slug,
       })
 
-      ;['createdBy', 'updatedBy', 'publishedAt', 'id', 'createdAt'].forEach(
-        (key) => {
-          if (!initialData[key]) return
-          translatedData[key] = initialData[key]
-        }
-      )
+      const parsedData = parseRelations(translatedData, allLayoutData)
 
-      // FIXME: Two issues here
-      // - Date/time field is only shown with value after save
-      // - The dispatch updates not just modified data but also the initial data
-      //   -> Saving is impossible until manual modification if object already existed
-      dispatch({
-        type: 'ContentManager/CrudReducer/GET_DATA_SUCCEEDED',
-        data: translatedData,
+      console.log(parsedData)
+      ;[
+        'createdBy',
+        'updatedBy',
+        'publishedAt',
+        'id',
+        'createdAt',
+        'updatedAt',
+      ].forEach((key) => {
+        _.omit(parsedData, key)
+
+        if (!initialData[key]) return
+        parsedData[key] = initialData[key]
       })
 
+      for (const key in parsedData) {
+        if (Object.hasOwnProperty.call(parsedData, key)) {
```

Comment:
is there a reason for the .call instead of a regular call (i.e. `Object.hasOwnProperty(parsedData, key)`)?

Reply:
That was the suggestion of a VSCode snippet.

Doing what you suggest creates an eslint error: `Do not access Object.prototype method 'hasOwnProperty' from target object eslint[no-prototype-builtins](https://eslint.org/docs/rules/no-prototype-builtins)`

Reply:
Ok I understand. ´Object.hasOwnProperty(parsedData, key)´ would actually not have worked either way 😅